### PR TITLE
Use MIT License for non Windows builds

### PR DESCRIPTION
**Pull request by ellismg**
The windows zips and installer still contain a few non MIT licensed
components, so we continue to use the .NET Library License for them.

### CODE REVIEW

**Comment by ellismg** [comment]
@richlander This should do what we discussed.  The ZIPs will now have the MIT licenses in them, as well as the OSX PKGs.

All windows artifacts contain artifacts licensed with the MSFT license.

/cc @omajid

**Comment by rakeshsinghranchi** [comment]
@ellismg , as per https://github.com/dotnet/core-setup/issues/2516 and https://github.com/dotnet/core-setup/issues/2513 , Windows and OSX installer do not show Eula screen. Instead, MIT License link is shown in the welcome page. With your change, do you plan to bring back the Eula in the UI ?

**Comment by ellismg** [comment]
@rakeshsinghranchi I have no plans to make any changes to the UI in the OSX installers. @richlander is that something we should be doing now that things are MIT licensed?

File: src/pkg/packaging/windows/host/generatemsi.ps1 [hunk 1]
```diff
@@ -35,7 +35,7 @@ function RunCandle
         -out "$WixObjRoot\" `
         -ext WixDependencyExtension.dll `
         -dHostSrc="$SharedHostPublishRoot" `
-        -dMicrosoftEula="$PackagingRoot\osx\sharedhost\resources\en.lproj\eula.rtf" `
+        -dMicrosoftEula="$PackagingRoot\osx\license\msft\resources\en.lproj\eula.rtf" `
```
Comment (eerhardt) [inline src/pkg/packaging/windows/host/generatemsi.ps1]: Does it still make sense for the Windows installer to get the license from `osx`, now that they aren't shared anymore?

Would it make more sense to put the msft license in the `packaging/windows` folder now?

Comment (ellismg) [inline src/pkg/packaging/windows/host/generatemsi.ps1]: I can do that.

**Pull request merged**